src/client/utils/mapper.js

```diff
@@ -0,0 +1,519 @@
+    receivesVaPension: null, // TODO - Mehedi claims this doesn't map to ES
+  },
+
+  additionalInformation: {
+    facilityState: null, // not used by ES
+  },
+
+  demographicInformation: {
+  },
+
+  veteranAddress: {
+    emailConfirmation: null, // not used by ES
+  },
+
+  financialDisclosure: {
+    provideFinancialInfo: false, // not used by ES
+    understandsFinancialDisclosure: false, // not used by ES
+  },
+
+  spouseInformation: {
+    dateOfMarriage: { // TODO not in ES
+      month: null,
+      day: null,
+      year: null
+    },
+  },
+
+  childInformation: {
+    hasChildrenToReport: false, // not directly used by ES
+    children: [blankChildInformation],
+  },
+
+  annualIncome: {
+  },
+
+  deductibleExpenses: {
+  },
+
+  insuranceInformation: {
+    isCoveredByHealthInsurance: false, // not used by ES
+  },
+
+  medicareMedicaid: { // this and insuranceInformation.providers[] populates insuranceList
+    isMedicaidEligible: false, // not used by ES
+  },
+
+  serviceInformation: {
+  },
+
+  militaryAdditionalInfo: {
+    postNov111998Combat: false, // TODO awaiting info from ES
+    disabledInLineOfDuty: false, // TODO awaiting info from ES
+  }
+};
+
+*/
+
+function makeDateString(dateMap) {
+  return `${dateMap.month}/${dateMap.day}/${dateMap.year}`;
+}
+
+function makeSSN(ssn) {
+  return ssn.replace('-', '');
+}
+
+function makeGender(gender) {
+  // There are non-active gender classifications in VHA Standard Data Service but
+  // M/F are the only active values as of early 2016
+  return gender;
+}
+
+function makeRaces(demographicInformation) {
+  // from VHA Standard Data Service (ADRDEV01) HL7 24 Race Map List
+  const races = [];
+  if (demographicInformation.isAmericanIndianOrAlaskanNative) races.push('1002-5');
+  if (demographicInformation.isAsian) races.push('2028-9');
+  if (demographicInformation.isBlackOrAfricanAmerican) races.push('2054-5');
+  if (demographicInformation.isNativeHawaiianOrOtherPacificIslander) races.push('2076-8');
+  if (demographicInformation.isWhite) races.push('2106-3');
+  return races;
+}
+
+function makeEthnicity(demographicInformation) {
+  // from VHA Standard Data Service (ADRDEV01) HL7 24 Ethnicity Map List
+  switch (demographicInformation.isSpanishHispanicLatino) {
+    case true:
+      return '2135-2';
+    case false:
+      return '2186-5';
+    default:
+      return '0000-0';
+  }
+}
+
+function makeMaritalStatus(maritalStatus) {
+  // from VHA Standard Data Service (ADRDEV01) HL7 24 Marital Status Map List
+  switch (maritalStatus) {
+    case 'Married':
+      return 'M';
+    case 'Never Married':
+      return 'S';
+    case 'Separated':
+      return 'A';
+    case 'Widowed':
+      return 'W';
+    case 'Divorced':
+      return 'D';
+    default:
+      return 'U';
+  }
+}
+
+function makeSpouse(nameAndGeneralInformation, spouseInformation) {
+  if (nameAndGeneralInformation.maritalStatus !== 'Never Married') {
```

Comment:
why not do 
if (nameAndGeneralInformation.maritalStatus === 'Never Married) {
  return undefined; // also why undefined? is there a reason?
}
return {....